doc/lua_api.txt

```diff
@@ -6915,10 +6915,28 @@ child will follow movement and rotation of that bone.
     object.
 * `set_detach()`
 * `set_bone_position([bone, position, rotation])`
-    * `bone`: string. Default is `""`, the root bone
-    * `position`: `{x=num, y=num, z=num}`, relative, `default {x=0, y=0, z=0}`
-    * `rotation`: `{x=num, y=num, z=num}`, default `{x=0, y=0, z=0}`
+	* Shorthand for `set_bone_override(bone, {position = ..., rotation = ...})` using absolute values.
+	* Supports older servers & clients (< 5.6)
+	* **Deprecated**; use `set_bone_override` instead
 * `get_bone_position(bone)`: returns position and rotation of the bone
+	* Shorthand for `get_bone_override(bone).position.vector, get_bone_override(bone).rotation.vector`
+	* **Deprecated**, use `get_bone_override` instead
+* `set_bone_override(bone, override)`
+    * `bone`: string
+    * `override`: `{ position = property, rotation = property, scale = property }` or `nil`
+        * `property`: `{ x = ..., y = ..., z = ..., interpolation_duration = 0, absolute = false}` or `nil`;
```

Comment:
I think something like `interpolate_dur` or just `interpolate` would be better

--

Also mixing tables too much is annoying.
Any reason not to use `{ pos = {...}, interpolate = 0, absolute = false }`?

Edit: This is double bad because the engine will return a vector with a metatable set and everything but also contains foreign fields. Definitely needs changing.